### DISCUSSION TITLE
Dev Chat.

### DISCUSSION THREAD

**Discussion by SoffiMeower** [obhq/obliteration#41] [General]
Have something you added to the project you wanted to talk about? Put it here!

**Comment by SoffiMeower** [comment]
Added a full version of the Obliteration logo! :3
![logo](https://user-images.githubusercontent.com/45863583/199580922-bc0258a4-d658-4cf7-b17a-d5b6fc611612.png)

**Reply by ultimaweapon** [reply]
I find this hard to read in dark mode. Maybe changing style for remaining characters after the first O?

**Reply by SoffiMeower** [reply]
I can try brightening the red! :3

**Reply by SoffiMeower** [reply]
How about now? Look at the readme.

**Reply by ultimaweapon** [reply]
Should be okay.

**Comment by SoffiMeower** [comment]
Did some changes to the release.yml in 8b3cbff64e4dd3159ca334e57c34ebe4bd8ea7b5 @ultimaweapon Tell me what you think! In order to cut down on YML size, I made it so the matrix names were used to set the presets on build! 
Plus it was still using old build files that no longer existed, like the VCPKG .sh and .bat files.

**Reply by ultimaweapon** [reply]
I'll take a look when we are going to make a release.

**Comment by SoffiMeower** [comment]
As it seems to be the pinnacle of 'first game working' for PS4 emulators... Gonna buy and dump a copy of We Are Doomed haha.

**Comment by SoffiMeower** [comment]
Also, Ultima, just had a guy try to use a PS4 updat file... Then he tried decrypting it. Would an extracted update technically 'work'?
https://www.psxhax.com/threads/ps4-pup_decrypt-pup_unpack-decrypt-unpack-ps4-updates-by-idc.1602/

**Reply by ultimaweapon** [reply]
No. The SPRX files are encrypted.

**Reply by SoffiMeower** [reply]
Exactly what I told him, just was wondering.

**Comment by SoffiMeower** [comment]
What do you think about the Wiki? Made it only have steps for Goldhen for now as that's the most popular method, but I'll add the FTP Payload bin later.

**Reply by ultimaweapon** [reply]
Let's see if it can help the people.

**Comment by red-prig** [comment]
What happened to the discord channel?

**Reply by ultimaweapon** [reply]
No idea. @VocalFan or @SuchAFuriousDeath should be able to answer.

**Reply by SuchAFuriousDeath** [reply]
I don't know. It just disappeared

**Reply by ultimaweapon** [reply]
You mean the whole Obliteration server disappeared?

**Reply by SuchAFuriousDeath** [reply]
yup

**Reply by ultimaweapon** [reply]
Seems like the only one who can answer is @VocalFan.

**Reply by SoffiMeower** [reply]
...Wait, so you're telling me it didn't stay even though I gave someone else owner role when I remade my account- Uh... Damn...

**Reply by red-prig** [reply]
wat

**Reply by SoffiMeower** [reply]
@red-prig Wanted to make a new account/rebrand, thought I gave the server to SAFD but I Discord decided otherwise?

**Reply by red-prig** [reply]
I don't know the server just disappeared and your account is called delete_user

**Reply by SoffiMeower** [reply]
Well, yeah, I wanted to make a new account (i hit the server limit lol), so I passed ownership of the server to SuchAFD, but I guess Discord went "nope"

**Reply by ultimaweapon** [reply]
So what will we do next?

**Reply by SoffiMeower** [reply]
I was thinking... @ultimaweapon Wanna hold off on the Discord until we get a game running? Most of the questions were asking when we'd be running games and well, that's... Impossible to guess properly.

**Reply by polybiusproxy** [reply]
> I was thinking... @ultimaweapon Wanna hold off on the Discord until we get a game running? Most of the questions were asking when we'd be running games and well, that's... Impossible to guess properly.

I know I'm no one to decide that, but I believe a server would still be useful for development purposes

**Reply by ultimaweapon** [reply]
@SuchAFuriousDeath do you want to make one for development?

**Reply by ultimaweapon** [reply]
Or @mikusp? If both of you don't want to handle this I'll looking for a different solution for development discussion.

**Reply by SoffiMeower** [reply]
Currently, we use github discussion and issues, no? The server was meant more for community anyways and well... We only really had 1 question a day and it was asking when we'd have it done or if X game would work. That's why I'm thinking to hold off on Discord until we are... Y'know... Operational in ONE game.

**Reply by ultimaweapon** [reply]
The discussion in question is for development, not for the end-users.

**Reply by ultimaweapon** [reply]
Here we are: https://obkrnl.zulipchat.com

**Comment by SoffiMeower** [comment]
Oh uh, hmm.

"You need an invitation to join this organization."

**Reply by ultimaweapon** [reply]
Fixed.
